spec/models/assignment_spec.rb

```diff
@@ -1026,7 +1025,11 @@
           result = s.get_latest_result
           result.total_mark = total_mark
           result.marking_state = Result::MARKING_STATES[:complete]
-          result.save
+          @assignment.rubric_criteria.each do |cri|
+            result.marks.create!(markable_id: cri.id, markable_type: RubricCriterion, mark: (total_mark*4.0/20).round)
```

Comment:
Line is too long. [118/80]<br>Surrounding space missing for operator `/`.<br>Surrounding space missing for operator `*`.